packages/zoe/src/zoeService/startInstance.js

```diff
@@ -81,7 +81,12 @@ export const makeStartInstance = (
           zoeSeatAdmins.forEach(zoeSeatAdmin => zoeSeatAdmin.fail(reason));
         },
         stopAcceptingOffers: () => (acceptingOffers = false),
-        makeUserSeat: (invitationHandle, initialAllocation, proposal) => {
+        makeUserSeat: (
+          invitationHandle,
+          initialAllocation,
+          proposal,
+          offerArgs,
```

Comment:
```suggestion
          offerArgs = undefined,
```

Reply:
Good point, I'll go fix any cases that don't make it clear that it may be undefined.